packages/ui-core/src/components/Select/Select.test.tsx

```diff
@@ -25,19 +34,140 @@ const props = {
     ],
 };
 
-const controlSelector = '.mfui-select__control';
 describe('<Select />', () => {
-    it('it renders Select', () => {
-        const wrapper = mount(<Select items={props.items} />);
-        expect(wrapper).toMatchSnapshot();
+    describe('snapshots', () => {
+        it('it ren ders Select', () => {
+            const wrapper = mount(<Select items={props.items} />);
+            expect(wrapper).toMatchSnapshot();
+        });
+
+        it('it renders Select with props', () => {
```

Comment:
Выше в describe уже описано, что эти тесты относятся к селекту
```suggestion
        it('it renders with props', () => {
```